src/frontend/js/hooks/useDashboardRoutes.tsx

```diff
@@ -0,0 +1,38 @@
+import { useIntl } from 'react-intl';
+import { useMemo } from 'react';
+import slugify from 'slugify';
+import dashboardRoutesDefinition, { DashBoardRoute } from 'components/DashBoard/routes';
+import { useSession } from 'data/SessionProvider';
+import ProtectedRoute from 'components/DashBoard/ProtectedRoute';
+
+const useDashBoardRoutes = () => {
+  const intl = useIntl();
+
+  const { user } = useSession();
+
+  const routes = useMemo<Array<DashBoardRoute>>(() => {
+    return dashboardRoutesDefinition.map((dashboardRoute) => ({
```

Comment:
Since we renamed `dashboardRoutes` into `dashboardRoutesDefinition`, I think `dashboardRoute` should be renamed.
```suggestion
    return dashboardRoutesDefinition.map((routeDefinition) => ({
```